packages/portal/src/plugins/europeana/set.js

```diff
@@ -32,6 +32,10 @@ export default class EuropeanaSetApi extends EuropeanaApi {
       if (params.profile === 'items.meta' && options.withMinimalItemPreviews === true) {
         params.profile = 'standard';
       }
+      // check for items.description profile
+      if (params.profile === 'items.meta') {
+        params.profile = 'itemDescriptions';
+      }
```

Comment:
This is going to override the previous condition (added as part of https://europeana.atlassian.net/browse/EC-7058). Perhaps move this condition above that?

Reply:
If the "withMinimalItemPreviews" condition is true it will switch to the standard profile and this won't trigger. Which is fine.

Reply:
Indeed. I have rewritten the conditions to make this clearer and DRYer.